go/vt/topo/helpers/copy.go

```diff
@@ -85,51 +70,34 @@ func CopyShards(ctx context.Context, fromTS, toTS *topo.Server) {
 		log.Fatalf("fromTS.GetKeyspaces: %v", err)
 	}
 
-	wg := sync.WaitGroup{}
-	rec := concurrency.AllErrorRecorder{}
 	for _, keyspace := range keyspaces {
-		wg.Add(1)
-		go func(keyspace string) {
-			defer wg.Done()
-			shards, err := fromTS.GetShardNames(ctx, keyspace)
+		shards, err := fromTS.GetShardNames(ctx, keyspace)
+		if err != nil {
+			log.Fatalf("GetShardNames(%v): %v", keyspace, err)
+			return
+		}
+
+		for _, shard := range shards {
```

Comment:
One issue with this loop is that it won't remove additional shards in the target, but I think it's acceptable. The main use case we want to cover is a fresh copy.

Reply:
Yes, I agree. I think this tool should be used only in the context of a fresh copy. In the future we could change it to be sync topos.